common/network-common/src/main/java/org/apache/spark/network/util/TransportFrameDecoder.java

```diff
@@ -48,14 +49,29 @@
   private static final int LENGTH_SIZE = 8;
   private static final int MAX_FRAME_SIZE = Integer.MAX_VALUE;
   private static final int UNKNOWN_FRAME_SIZE = -1;
+  private static final long DEFAULT_CONSOLIDATE_FRAME_BUFS_DELTA_THRESHOLD = 20 * 1024 * 1024;
```

Comment:
Sorry but I can't really decode the name of this constant, nor of the fields that relate to it.

What is "consolidate frame bufs delta"?

Reply:
What's your suggestion? I'm really torn on this name, what I mean is that if the delta size of input data since last consolidation exceeds the threshold, then the frameBuf should do consolidation.

Reply:
`CONSOLIDATE_THRESHOLD` or something?

Reply:
Thanks, @srowen , done!